article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -146,89 +159,106 @@ of the jacobian derivation see supplementary_notebook_2.ipynb). As an expansion
 (this model is already implemented in Dipy).
 
 # Results
 
-The results from the Monte Carlo simulations are shown in Figure @fig:simulations. Similarly to what is reported
+The results from the Monte Carlo simulations are shown in Figure @fig:simulations. AS reported similarly
 in the original article, FA values estimated using the free water elimination model match the tissue's ground truth
-values for free water volume fractions $f$ ranging around 0 to 0.7 (top panel of
+values for free water volume fractions $f$ ranging from 0 to 0.7 (top panel of
 Figure @fig:simulations). However, FA values seem to be overestimated for higher volume fractions. This bias is more
-prominent for lower FA values in which overestimations are visible from lower free water volume
+prominent for lower FA values, in which overestimation is visible from lower free water volume
 fractions. The lower panels of Figure @fig:simulations suggest that the free water elimination model produce
 accurate free water volume fraction for the full range of volume fraction ground truth values. All the features observed
 here are consistent with Fig.5 of the original article.
 
-![Fractional Anisotropy (FA) and free water volume fraction ($f$) estimates obtained with the from the Monte Carlo simulations
-using the free water elimination fitting procedures. The top panel shows the FA median and intra-quartil range
-for the five different FA ground truth levels and plotted as function of the ground truth water volume fraction.
-The bottom panels show the estimated volume fraction $f$ median and intra-quartil range as function of its ground truth values
+![Fractional Anisotropy (FA) and free water volume fraction ($f$) estimates obtained with Monte Carlo simulations
+using the free water elimination fitting procedures. The top panel shows the FA median and interquartile range
+for the five different FA ground truth levels plotted as function of the ground truth water volume fraction.
+The bottom panels show the estimated volume fraction $f$ median and interquartile range as function of its ground truth values
 (right and left panels correspond to the higher and lower FA values, respectively). This figure reproduces
 Fig.7 of the original article](fwdti_simulations.png){#fig:simulations}
 
 In vivo tensor statistics obtained from the free water elimination and standard DTI models
 are shown in Figure @fig:invivo. Complete processing of all these measure took less than 1 hour
-in an average Desktop and Laptop PC (~2GHz processor speed), while the reported processing time
-by Hoy et al. was around 20hours. The free water elimination model seems to produce higher values
+in an average desktop and laptop PC (~2GHz processor speed (how many cores, parallel or not?)), 
+hile the reported processing time by Hoy et al. was around 20 hours. 
+
+sentence to say this is a ballpark estimate, since they used a different numerical (or not) fitting, different platform, code, dataset, etc.
```

Comment:
I know it's faster, but since they used a completely different setup it almost sounds like slander. If the poor guys redid a newton method by themselves, no wonder it is 20 times slower though. Do they mention their dataset size or computer spec/language?